doc/Programming-guide/Key-platform-features/Data-collection/index.md

```diff
@@ -5,4 +5,1175 @@ permalink: /:path/
 nav: /:path/Programming-guide/Key-platform-features/Data-collection
 sort_idx: 50
 ---
-Purpose of data collection. Schemas and versioning. Store and forward (EP-side caching). Log appenders. Reliability. Sequencing. SDK-side API, log storage, log upload triggers. Avro wrapper schema.
+
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+* TOC
+{:toc}
+
+The Kaa data collection subsystem is designed to collect records (logs) of pre-configured structure on the endpoints, periodically transferring these records
+from endpoints to Operation services, and, finally, either persisting them on the server for further processing or submitting them to immediate stream
+analysis. The log structure in Kaa is determined for each application by a configurable log schema. On the Operation service side, there are log appenders
+which are responsible for writing logs received by the Operations service into the specific storage.
```

Comment:
Is it possible to have more than one log appender working simultaneously? If yes, could this information be added?